scripts/cem-plugin-react-wrapper.js

```diff
@@ -0,0 +1,252 @@
+                        m.path.indexOf('sync/') === -1
+                )
+                .map(
+                    (m) =>
+                        `import '${pkgName}/${m.path?.replace('.ts', '.js')}';`
+                );
+
+            const reactComponents = [];
+
+            for (let component of components) {
+                componentImports.push(
+                    `import { ${component.name} as Sp${component.name} } from '${pkgName}';`
+                );
+
+                const reactComponent = {};
+                reactComponent.name = `${component.name}`;
+                reactComponent.swcComponentName = `Sp${component.name}`;
+                reactComponent.elementName = component.tagName;
+                const events = [];
+                await getEvents(component, declMap, events);
+                reactComponent.events = uniqueBy(events.flat(), 'name');
+
+                reactComponents.push(reactComponent);
+            }
+
+            const componentSrc = `/*
+Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import * as React from 'react';
+import { createComponent } from '@lit-labs/react';
+${componentImports.reduce((pre, cur) => pre + cur + '\n', '')}
+${fileImports.reduce((pre, cur) => pre + cur + '\n', '')}
+
+${reactComponents.reduce(
+    (pre, component) =>
+        pre +
+        `export const ${component.name} = createComponent({
+        displayName: '${component.name}',
+        elementClass: ${component.swcComponentName},
+        react: React,
+        tagName: '${component.elementName}',
+        events: {
+            ${component.events.reduce(
+                (pre, event) =>
+                    pre +
+                    `${event.name.replace(/-./g, (m) =>
+                        m[1].toUpperCase()
+                    )}: '${event.name}', `,
+                ''
+            )}
+        }
+    });`,
+    ''
+)}
+`;
+
+            const componentShortName = pkgName.replace(
+                '@spectrum-web-components/',
+                ''
+            );
+
+            const packageJson = `{
+    "name": "@swc-react/${componentShortName}",
+    "version": "${pkgVersion}",
+    "publishConfig": {
+        "access": "public"
+    },
+    "description": "React wrapper of the ${pkgName} component",
+    "license": "Apache-2.0",
+    "author": "",
+    "main": "index.js",
+    "module": "index.js",
+    "type": "module",
+    "keywords": [
```

Comment:
You mentioned that this initial releases worked in TS but not in JS, and I think that points to us leaking some implementation defaults that we don't surface in the main packages. With that in mind, can we includes a `"files"` listing that is similar to what we see in those packages, here?
```suggestion
    "files": [
        "**/*.d.ts",
        "**/*.js",
        "**/*.js.map"
    ],
    "keywords": [
```

Reply:
Done